src/main/scala/com/fulcrumgenomics/phasing/HapCutToVcf.scala

```diff
@@ -0,0 +1,413 @@
+import htsjdk.variant.vcf._
+
+import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+
+@clp(
+  description =
+    """
+      |Converts the output of HapCut to a VCF.
+      |
+      |The output of HAPCUT does not include all input variants, but simply those variants that are in phased blocks.
+      |This tool takes the original VCF and the output of HapCut, and produces two VCFs: (1) a VCF containing variants
+      |that were phased, and (2) a VCF containing variants that were not phased, such that all variants in the original
+      |file are in one of two output VCFs.
+      |
+      |The VCF with phased variants will have the first variant of a haplotype block with a genotype "M\F", while the
+      |subsequent variants in that block having "M|F" as their genotype.  This first variant in a block can optionally
+      |have "M|F".
+      |
+    """,
+  group=ClpGroups.Phasing
+)
+class HapCutToVcf
+( @arg(flag="v", doc="The original VCF provided to HAPCUT.") val vcf: PathToVcf,
+  @arg(flag="i", doc="The output file from HAPCUT.") val input: FilePath,
+  @arg(flag="o", doc="The output VCF with variants that were phased by HAPCUT.") val output: PathToVcf,
+  @arg(flag="u", doc="The output VCF with variants that were not phased by HAPCUT.") val unphasedOutput: PathToVcf,
+  @arg(flag="m", doc="Allow missing fields in the VCF header.") val allowMissingFieldsInVcfHeader: Boolean = true,
+  @arg(flag="r", doc="The first variant context in a block should have '/' instead of '|'") val readBackedPhasingFormat: Boolean = true
+) extends FgBioTool with LazyLogging {
+  Io.assertReadable(vcf)
+  Io.assertReadable(input)
+  Io.assertCanWriteFile(output)
+
+  override def execute(): Unit = {
+    val vcfReader      = new VCFFileReader(vcf.toFile, false)
+    val phasedWriter   = makeWriter(output, vcfReader, addPhasedHeaderLines=true)
+    val unphasedWriter = makeWriter(unphasedOutput, vcfReader, addPhasedHeaderLines=false)
+
+    val iterator = new MergingIterator(input, vcfReader, readBackedPhasingFormat)
+    iterator.foreach {
+      case Left(ctx)  => phasedWriter.add(ctx)
+      case Right(ctx) => unphasedWriter.add(ctx)
+    }
+
+    CloserUtil.close(vcfReader)
+    phasedWriter.close()
+    unphasedWriter.close()
+  }
+
+  /** Creates a VCF writer, adding extra header lines if the output is the phased VCF. */
+  private def makeWriter(path: PathToVcf, vcfReader: VCFFileReader, addPhasedHeaderLines: Boolean): VariantContextWriter = {
+    val inputHeader = vcfReader.getFileHeader
+
+    val builder = new VariantContextWriterBuilder()
+      .setOutputFile(path.toFile)
+      .setReferenceDictionary(inputHeader.getSequenceDictionary)
+      .setOption(Options.INDEX_ON_THE_FLY)
+      .modifyOption(Options.ALLOW_MISSING_FIELDS_IN_HEADER, allowMissingFieldsInVcfHeader)
+    val writer: VariantContextWriter = builder.build
+
+    val outHeader = if (addPhasedHeaderLines) {
+      // get the header lines to skip
```

Comment:
`// get the header lines in the input header that we wish to skip/replace with our own definitions`